docs/en/ingest-management/elastic-agent/example-standalone-monitor-nginx-ess.asciidoc

```diff
@@ -143,7 +143,7 @@ image::images/guide-install-agent-on-host.png["Install {agent} on your host step
 +
 [NOTE] 
 ==== 
-{agent} commands need to be run as `root`. You can prefix each agent command with `sudo` or you can start a new shell as `root` by running `sudo su`. 
+It's recommended to run {agent} commands as `root`. You can prefix each agent command with `sudo` or you can start a new shell as `root` by running `sudo su`. For details about run {agent} commands without `root` access, refer to <<elastic-agent-unprivileged>>.
```

Comment:
What about something like "consider running agent commands..." to avoid starting a sentence with an orphaned pronoun?

Reply:
@karenzone Thanks for catching that. I've changed this to:

> {agent} commands should be run as `root`. You can prefix each agent command with `sudo` or you can start a new shell as `root` by running `sudo su`. If you need to run {agent} commands without `root` access, refer to <<elastic-agent-unprivileged>>.

What do you think? I didn't use 'consider' because the really should be run as root whenever possible.